GithubIssueTagger/Program.cs

```diff
@@ -1,85 +1,15 @@
-
         static async Task Main(string[] args)
         {
             var client = new GitHubClient(new ProductHeaderValue("nuget-github-issue-tagger"));
-            client.Credentials = new Credentials(GetClientSecret());
-
-            await RemoveLabelFromAllIssuesAsync(client, "ToBeMoved", "nuget", "home");
-
-            Predicate<Issue> predicate = WasUpdatedInLastSixMonths;
-            await AddLabelToMatchingIssues(client, "ToBeMoved", "nuget", "home", predicate);
-
-            Console.WriteLine("Done!");
-        }
-
-        private static string GetClientSecret()
-        {
-            return token;
-        }
-
-        private static bool WasUpdatedInLastSixMonths(Issue issue)
-        {
-            return issue.State == ItemState.Open && issue.UpdatedAt.Value.DateTime < SixMonthsFromAppStartup;
-        }
-
-        private static async Task AddLabelToMatchingIssues(GitHubClient client, string label, string org, string repo, Predicate<Issue> predicate)
-        {
-            var issuesForRepo = await client.Issue.GetAllForRepository(org, repo);
-
-            foreach (var issue in issuesForRepo)
-            {
-                if (predicate(issue))
-                {
-                    try
-                    {
-                        var issueUpdate = issue.ToUpdate();
-                        issueUpdate.AddLabel(label);
-                        await client.Issue.Update(org, repo, issue.Number, issueUpdate);
-                        Console.WriteLine($"Updated issue: {issue.HtmlUrl}");
-                    }
-                    catch (Exception e)
-                    {
-                        Console.WriteLine($"Unhandled issue {issue.HtmlUrl} {e}");
-                    }
-                }
-            }
-        }
-
-        private static async Task RemoveLabelFromAllIssuesAsync(GitHubClient client, string label, string org, string repo)
-        {
-            var issuesForRepo = await client.Issue.GetAllForRepository(org, repo);
-
-            foreach (var issue in issuesForRepo)
-            {
-                if (issue.Labels != null)
-                {
-                    if (issue.Labels.Any(e => e.Name.Equals(label)))
-                    {
-                        try
-                        {
-                            var issueUpdate = issue.ToUpdate();
-                            issueUpdate.RemoveLabel(label);
-                            await client.Issue.Update(org, repo, issue.Number, issueUpdate);
-                            Console.WriteLine($"Updated issue: {issue.HtmlUrl}");
-                        }
-                        catch (Exception e)
-                        {
-                            Console.WriteLine($"Unhandled issue {issue.HtmlUrl} {e}");
-                        }
-                    }
-                }
-            }
+            client.Credentials = new Credentials(args[0]);
```

Comment:
I know this is a non-production tool, but the most basic of arg checking would be nice :)

```cs
if (args.count != 1)
{
  Console.Err.WriteLine("Expected 1 argument (github PAT). Found " + args.Count);
  return;
}
```

Reply:
Add a suggestion and I'll merge it. 

I really don't want to be making changes :D

Reply:
Added it.